main.cpp

```diff
@@ -44,21 +44,19 @@ struct MyStruct1                                //4
             if( a->value > b->value ) return b;
```

Comment:
Did you evaluate if you can convert `a` or `b` into References?
- [ ] revise

Reply:
i did before but i guess i didn't try everything to get over the error.